tests/Controller/EnMarche/SummaryManagerControllerTest.php

```diff
@@ -608,7 +608,9 @@ public function testSearchSkillUserHasNot()
         $this->authenticateAsAdherent($this->client, 'luciole1989@spambox.fr', 'EnMarche2017');
 
         // Search the skill that user has not, should find one skill
-        $this->client->request(Request::METHOD_GET, 'espace-adherent/mon-profil/competences/autocompletion?term=outi');
+        $this->client->request(Request::METHOD_GET, '/espace-adherent/mon-profil/competences/autocompletion?term=outi', [], [], [
+            'HTTP_X-Requested-With' => 'XMLHttpRequest',
```

Comment:
Was it a false positive since this route only accepts XMLHttpRequests ?

Did the host declaration changed how the Client behaves for XMLHttpRequests?